ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_luxembourg_prod?event=eyJhY3Rpb24iOiJWaXNpdGVkIC0gc2VhcmNoLXJlc3VsdHMiLCJ1c2VyIjp7ImF0dHJpYnV0ZXMiOnsiZ2FDbGllbnRJRCI6IjYwNDU2NzIxMC4xNzQ2NDM3NjIwIn0sImFub25JZCI6ImRmZTlmODAxMjk0ZmFkMjEifSwic291cmNlIjp7InVybCI6Imh0dHBzOi8vd3d3Lm1lcmNlZGVzLWJlbnoubHUvZGUvcGFzc2VuZ2VyY2Fycy9idXkvbmV3LWNhci9zZWFyY2gtcmVzdWx0cy5odG1sL3ZlaGljbGVDYXRlZ29yeS1uZXctcGFzc2VuZ2VyLWNhcnMvYnJhbmQtTWVyY2VkZXMtQmVuei9tb2RlbElkZW50aWZpZXItQ0xBLz9lbWhzb3J0VHlwZT1wcmljZS1hc2MmZW1oYm9keVR5cGU9Q09VUEUiLCJ1cmxSZWZlcnJlciI6IiIsImNoYW5uZWwiOiJXZWIiLCJiZWFjb25WZXJzaW9uIjoxNiwiY29uZmlnVmVyc2lvbiI6IjU0IiwiY29udGVudFpvbmVzIjpbXX0sImNvbnNlbnRzIjpbXSwiYWNjb3VudCI6e30sIl90b29sc0V2ZW50TGlua0lkIjoiODE2ODkyNDM0NjMwNzMwNSIsImRlYnVnIjp7ImV4cGxhbmF0aW9ucyI6dHJ1ZX19: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.114)
Stacktrace:
#0 0x55666866575a <unknown>
#1 0x5566681184b0 <unknown>
#2 0x5566680fed86 <unknown>
#3 0x5566680fd69a <unknown>
#4 0x5566680fddff <unknown>
#5 0x5566680fdd54 <unknown>
#6 0x55666811c69f <unknown>
#7 0x5566681c192a <unknown>
#8 0x55666818f592 <unknown>
#9 0x5566681b55e0 <unknown>
#10 0x55666818f363 <unknown>
#11 0x55666815bd63 <unknown>
#12 0x55666815c9c1 <unknown>
#13 0x55666862aa6b <unknown>
#14 0x55666862e951 <unknown>
#15 0x556668612b62 <unknown>
#16 0x55666862f4c4 <unknown>
#17 0x5566685f713f <unknown>
#18 0x5566686536f8 <unknown>
#19 0x5566686538d6 <unknown>
#20 0x5566686645a6 <unknown>
#21 0x7fd0aa09caa4 <unknown>
#22 0x7fd0aa129c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_luxembourg_prod?event=eyJpdGVtQWN0aW9uIjpudWxsLCJzb3VyY2UiOnsicGFnZVR5cGUiOiJkZWZhdWx0IiwibG9jYWxlIjoiZGVfTFUiLCJjb250ZW50Wm9uZXMiOltdLCJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56Lmx1L2RlL3Bhc3NlbmdlcmNhcnMvYnV5L25ldy1jYXIvc2VhcmNoLXJlc3VsdHMuaHRtbC92ZWhpY2xlQ2F0ZWdvcnktbmV3LXBhc3Nlbmdlci1jYXJzL2JyYW5kLU1lcmNlZGVzLUJlbnovbW9kZWxJZGVudGlmaWVyLUNMQS8%2FZW1oc29ydFR5cGU9cHJpY2UtYXNjJmVtaGJvZHlUeXBlPUNPVVBFIiwidXJsUmVmZXJyZXIiOiIiLCJjaGFubmVsIjoiV2ViIiwiYmVhY29uVmVyc2lvbiI6MTYsImNvbmZpZ1ZlcnNpb24iOiI1NCJ9LCJmbGFncyI6eyJwYWdlVmlldyI6dHJ1ZX0sInVzZXIiOnsiYXR0cmlidXRlcyI6eyJnYUNsaWVudElEIjoiNjA0NTY3MjEwLjE3NDY0Mzc2MjAifSwiYW5vbklkIjoiZGZlOWY4MDEyOTRmYWQyMSJ9LCJwZXJmb3JtYW5jZSI6eyJzZGtQYXJzZVRpbWUiOjJ9LCJkZWJ1ZyI6eyJleHBsYW5hdGlvbnMiOnRydWV9LCJjYXRhbG9nIjp7fSwiY29uc2VudHMiOltdLCJhY2NvdW50Ijp7fSwiX3Rvb2xzRXZlbnRMaW5rSWQiOiI4NTc4Mzc2MzQ1NTYzNzM0In0%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.114)
Stacktrace:
#0 0x55666866575a <unknown>
#1 0x5566681184b0 <unknown>
#2 0x5566680fed86 <unknown>
#3 0x5566680fd69a <unknown>
#4 0x5566680fddff <unknown>
#5 0x5566680fdd54 <unknown>
#6 0x55666811c69f <unknown>
#7 0x5566681c192a <unknown>
#8 0x55666818f592 <unknown>
#9 0x5566681b55e0 <unknown>
#10 0x55666818f363 <unknown>
#11 0x55666815bd63 <unknown>
#12 0x55666815c9c1 <unknown>
#13 0x55666862aa6b <unknown>
#14 0x55666862e951 <unknown>
#15 0x556668612b62 <unknown>
#16 0x55666862f4c4 <unknown>
#17 0x5566685f713f <unknown>
#18 0x5566686536f8 <unknown>
#19 0x5566686538d6 <unknown>
#20 0x5566686645a6 <unknown>
#21 0x7fd0aa09caa4 <unknown>
#22 0x7fd0aa129c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_luxembourg_prod?event=eyJhY3Rpb24iOiJTZWFyY2ggUmVzdWx0IFBhZ2UgLSBDYXJzIiwidXNlciI6eyJhdHRyaWJ1dGVzIjp7ImdhQ2xpZW50SUQiOiI2MDQ1NjcyMTAuMTc0NjQzNzYyMCJ9LCJhbm9uSWQiOiJkZmU5ZjgwMTI5NGZhZDIxIn0sInNvdXJjZSI6eyJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56Lmx1L2RlL3Bhc3NlbmdlcmNhcnMvYnV5L25ldy1jYXIvc2VhcmNoLXJlc3VsdHMuaHRtbC92ZWhpY2xlQ2F0ZWdvcnktbmV3LXBhc3Nlbmdlci1jYXJzL2JyYW5kLU1lcmNlZGVzLUJlbnovbW9kZWxJZGVudGlmaWVyLUNMQT9lbWhzb3J0VHlwZT1wcmljZS1hc2MmZW1oYm9keVR5cGU9Q09VUEUiLCJ1cmxSZWZlcnJlciI6IiIsImNoYW5uZWwiOiJXZWIiLCJiZWFjb25WZXJzaW9uIjoxNiwiY29uZmlnVmVyc2lvbiI6IjU0IiwiY29udGVudFpvbmVzIjpbXX0sImNvbnNlbnRzIjpbXSwiYWNjb3VudCI6e30sIl90b29sc0V2ZW50TGlua0lkIjoiNzA3MDI3OTAyMjkzNjY0MiIsImRlYnVnIjp7ImV4cGxhbmF0aW9ucyI6dHJ1ZX19: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.114)
Stacktrace:
#0 0x55666866575a <unknown>
#1 0x5566681184b0 <unknown>
#2 0x5566680fed86 <unknown>
#3 0x5566680fd69a <unknown>
#4 0x5566680fddff <unknown>
#5 0x5566680fdd54 <unknown>
#6 0x55666811c69f <unknown>
#7 0x5566681c192a <unknown>
#8 0x55666818f592 <unknown>
#9 0x5566681b55e0 <unknown>
#10 0x55666818f363 <unknown>
#11 0x55666815bd63 <unknown>
#12 0x55666815c9c1 <unknown>
#13 0x55666862aa6b <unknown>
#14 0x55666862e951 <unknown>
#15 0x556668612b62 <unknown>
#16 0x55666862f4c4 <unknown>
#17 0x5566685f713f <unknown>
#18 0x5566686536f8 <unknown>
#19 0x5566686538d6 <unknown>
#20 0x5566686645a6 <unknown>
#21 0x7fd0aa09caa4 <unknown>
#22 0x7fd0aa129c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_luxembourg_prod?event=eyJpdGVtQWN0aW9uIjpudWxsLCJzb3VyY2UiOnsicGFnZVR5cGUiOiJzZWFyY2hfcmVzdWx0IiwibG9jYWxlIjoiZGVfTFUiLCJjb250ZW50Wm9uZXMiOltdLCJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56Lmx1L2RlL3Bhc3NlbmdlcmNhcnMvYnV5L25ldy1jYXIvc2VhcmNoLXJlc3VsdHMuaHRtbC92ZWhpY2xlQ2F0ZWdvcnktbmV3LXBhc3Nlbmdlci1jYXJzL2JyYW5kLU1lcmNlZGVzLUJlbnovbW9kZWxJZGVudGlmaWVyLUNMQT9lbWhzb3J0VHlwZT1wcmljZS1hc2MmZW1oYm9keVR5cGU9Q09VUEUiLCJ1cmxSZWZlcnJlciI6IiIsImNoYW5uZWwiOiJXZWIiLCJiZWFjb25WZXJzaW9uIjoxNiwiY29uZmlnVmVyc2lvbiI6IjU0In0sImZsYWdzIjp7InBhZ2VWaWV3Ijp0cnVlfSwidXNlciI6eyJhdHRyaWJ1dGVzIjp7ImdhQ2xpZW50SUQiOiI2MDQ1NjcyMTAuMTc0NjQzNzYyMCJ9LCJhbm9uSWQiOiJkZmU5ZjgwMTI5NGZhZDIxIn0sInBlcmZvcm1hbmNlIjp7fSwiZGVidWciOnsiZXhwbGFuYXRpb25zIjp0cnVlfSwiY2F0YWxvZyI6e30sImNvbnNlbnRzIjpbXSwiYWNjb3VudCI6e30sIl90b29sc0V2ZW50TGlua0lkIjoiODA1MjYyOTc1Mzk1MDg5NiJ9: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.114)
Stacktrace:
#0 0x55666866575a <unknown>
#1 0x5566681184b0 <unknown>
#2 0x5566680fed86 <unknown>
#3 0x5566680fd69a <unknown>
#4 0x5566680fddff <unknown>
#5 0x5566680fdd54 <unknown>
#6 0x55666811c69f <unknown>
#7 0x5566681c192a <unknown>
#8 0x55666818f592 <unknown>
#9 0x5566681b55e0 <unknown>
#10 0x55666818f363 <unknown>
#11 0x55666815bd63 <unknown>
#12 0x55666815c9c1 <unknown>
#13 0x55666862aa6b <unknown>
#14 0x55666862e951 <unknown>
#15 0x556668612b62 <unknown>
#16 0x55666862f4c4 <unknown>
#17 0x5566685f713f <unknown>
#18 0x5566686536f8 <unknown>
#19 0x5566686538d6 <unknown>
#20 0x5566686645a6 <unknown>
#21 0x7fd0aa09caa4 <unknown>
#22 0x7fd0aa129c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_luxembourg_prod?event=eyJhY3Rpb24iOiJzZXRfdXNlcl9hdHRyaWJ1dGUiLCJhY2NvdW50Ijp7ImF0dHJpYnV0ZXMiOnsibGFzdFNSUE1vZGVsRmlsdGVyIjoie1wiZGNwX2xhc3RfbW9kZWxmaWx0ZXJcIjpcImNsYS1jbGFzc1wiLFwiZGNwX2xhc3RfYm9keXR5cGVmaWx0ZXJcIjpcImNvdXBlXCIsXCJkY3BfbGFzdF9zZWFyY2hfdXJsXCI6XCJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56Lmx1L2RlL3Bhc3NlbmdlcmNhcnMvYnV5L25ldy1jYXIvc2VhcmNoLXJlc3VsdHMuaHRtbC92ZWhpY2xlQ2F0ZWdvcnktbmV3LXBhc3Nlbmdlci1jYXJzL2JyYW5kLU1lcmNlZGVzLUJlbnovbW9kZWxJZGVudGlmaWVyLUNMQT9lbWhzb3J0VHlwZT1wcmljZS1hc2MmZW1oYm9keVR5cGU9Q09VUEVcIixcInRpbWVzdGFtcFwiOjE3NDY0Mzc2MjkzMTksXCJleHBpcmF0aW9uXCI6MTIwfSJ9fSwidXNlciI6eyJhdHRyaWJ1dGVzIjp7ImdhQ2xpZW50SUQiOiI2MDQ1NjcyMTAuMTc0NjQzNzYyMCJ9LCJhbm9uSWQiOiJkZmU5ZjgwMTI5NGZhZDIxIn0sInNvdXJjZSI6eyJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56Lmx1L2RlL3Bhc3NlbmdlcmNhcnMvYnV5L25ldy1jYXIvc2VhcmNoLXJlc3VsdHMuaHRtbC92ZWhpY2xlQ2F0ZWdvcnktbmV3LXBhc3Nlbmdlci1jYXJzL2JyYW5kLU1lcmNlZGVzLUJlbnovbW9kZWxJZGVudGlmaWVyLUNMQT9lbWhzb3J0VHlwZT1wcmljZS1hc2MmZW1oYm9keVR5cGU9Q09VUEUiLCJ1cmxSZWZlcnJlciI6IiIsImNoYW5uZWwiOiJXZWIiLCJiZWFjb25WZXJzaW9uIjoxNiwiY29uZmlnVmVyc2lvbiI6IjU0IiwiY29udGVudFpvbmVzIjpbXX0sImNvbnNlbnRzIjpbXSwiX3Rvb2xzRXZlbnRMaW5rSWQiOiI1MTMzODk2ODEzNzAxMzY3IiwiZGVidWciOnsiZXhwbGFuYXRpb25zIjp0cnVlfX0%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.114)
Stacktrace:
#0 0x55666866575a <unknown>
#1 0x5566681184b0 <unknown>
#2 0x5566680fed86 <unknown>
#3 0x5566680fd69a <unknown>
#4 0x5566680fddff <unknown>
#5 0x5566680fdd54 <unknown>
#6 0x55666811c69f <unknown>
#7 0x5566681c192a <unknown>
#8 0x55666818f592 <unknown>
#9 0x5566681b55e0 <unknown>
#10 0x55666818f363 <unknown>
#11 0x55666815bd63 <unknown>
#12 0x55666815c9c1 <unknown>
#13 0x55666862aa6b <unknown>
#14 0x55666862e951 <unknown>
#15 0x556668612b62 <unknown>
#16 0x55666862f4c4 <unknown>
#17 0x5566685f713f <unknown>
#18 0x5566686536f8 <unknown>
#19 0x5566686538d6 <unknown>
#20 0x5566686645a6 <unknown>
#21 0x7fd0aa09caa4 <unknown>
#22 0x7fd0aa129c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_luxembourg_prod?event=eyJhY3Rpb24iOiJWaXNpdGVkIC0gcHJvZHVjdCIsInVzZXIiOnsiYXR0cmlidXRlcyI6eyJnYUNsaWVudElEIjoiNjA0NTY3MjEwLjE3NDY0Mzc2MjAifSwiYW5vbklkIjoiZGZlOWY4MDEyOTRmYWQyMSJ9LCJzb3VyY2UiOnsidXJsIjoiaHR0cHM6Ly93d3cubWVyY2VkZXMtYmVuei5sdS9kZS9wYXNzZW5nZXJjYXJzL2J1eS9uZXctY2FyL3Byb2R1Y3QuaHRtbC9DTEEtMTgwLUNvdXBlLVN0YXItRWRpdGlvbl8wNTUxMzUzOTk3X0xVXzM1NGFlMmY1IiwidXJsUmVmZXJyZXIiOiIiLCJjaGFubmVsIjoiV2ViIiwiYmVhY29uVmVyc2lvbiI6MTYsImNvbmZpZ1ZlcnNpb24iOiI1NCIsImNvbnRlbnRab25lcyI6W119LCJjb25zZW50cyI6W10sImFjY291bnQiOnt9LCJfdG9vbHNFdmVudExpbmtJZCI6IjIxNDg4MzU2NDU0Mjk2MTc2IiwiZGVidWciOnsiZXhwbGFuYXRpb25zIjp0cnVlfX0%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.114)
Stacktrace:
#0 0x55666866575a <unknown>
#1 0x5566681184b0 <unknown>
#2 0x5566680fed86 <unknown>
#3 0x5566680fd69a <unknown>
#4 0x5566680fddff <unknown>
#5 0x5566680fdd54 <unknown>
#6 0x55666811c69f <unknown>
#7 0x5566681c192a <unknown>
#8 0x55666818f592 <unknown>
#9 0x5566681b55e0 <unknown>
#10 0x55666818f363 <unknown>
#11 0x55666815bd63 <unknown>
#12 0x55666815c9c1 <unknown>
#13 0x55666862aa6b <unknown>
#14 0x55666862e951 <unknown>
#15 0x556668612b62 <unknown>
#16 0x55666862f4c4 <unknown>
#17 0x5566685f713f <unknown>
#18 0x5566686536f8 <unknown>
#19 0x5566686538d6 <unknown>
#20 0x5566686645a6 <unknown>
#21 0x7fd0aa09caa4 <unknown>
#22 0x7fd0aa129c3c <unknown>

ERROR    root:VerifyPersonalizationAndCapture.py:18 ❌ Error attaching screenshot to Allure: [Errno 2] No such file or directory: '/home/runner/work/QA-Allure-Testing/QA-Allure-Testing/Tests/Last Seen PDP_attempt_2.png'